### PR TITLE
feat: create 2022-01-25-Community-Call-importanza-design-system

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

File: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Community Call: L’importanza di un design system per i servizi pubblici digitali"
```
Comment (bfabio) [inline _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:3]:
```suggestion
title: "Community call: L’importanza di un design system per i servizi pubblici digitali"
```

File: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+subtitle: Quali le sfide da affrontare? Ne parliamo il 9 febbraio!
+date: 2022-01-25 12:00:00 +0200
+tags: developers designers designsystem
+categories: news
+author: Michela Gabrieli
+image: /assets/icons/logo-it.png
+locale: it
+---
+
+Vi siete mai chiesti perché alcuni siti sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo. Per esempio i grandi e-commerce, i social network, il nostro sito preferito di streaming? Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente.
```
Comment (bfabio) [inline _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:13]: La prima frase non mi torna, dovrebbe avere un punto interrogativo.

Potrebbe essere:
> Vi siete mai chiesti perché alcuni siti, per esempio i grandi ecommerce o il vostro sito preferito di streaming, sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo?

File: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Non sarebbe bellissimo se ciò accadesse anche per tutti i siti e i servizi digitali della Pubblica Amministrazione?
+
+Proprio per rispondere a questo quesito, a partire dallo scorso dicembre [Designers Italia](https://designers.italia.it/) in collaborazione con Developers Italia ha avviato una revisione completa del design system del Paese, uno strumento utile a  creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. A breve sarà resa pubblica una roadmap di aggiornamento delle librerie dei componenti e dei [due modelli applicati](https://designers.italia.it/modelli/), [Comuni](https://designers.italia.it/modello/comuni/) e [scuole](https://designers.italia.it/modello/scuole/). Questa revisione affronta una serie di sfide peculiari, data la sua applicazione in ambito pubblico:
```
Comment (bfabio) [inline _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:17]: > in collaborazione con Developers Italia

Do we really want to keep Developers & Designers apart? Sounds too cold and formal to me

Comment (bfabio) [inline _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:17]:
```suggestion
Proprio per rispondere a questo quesito, a partire dallo scorso dicembre [Designers Italia](https://designers.italia.it/) in collaborazione con Developers Italia ha avviato una revisione completa del design system del Paese, uno strumento utile a  creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. A breve sarà resa pubblica una roadmap di aggiornamento delle librerie dei componenti e dei [due modelli applicati](https://designers.italia.it/modelli/), [comuni](https://designers.italia.it/modello/comuni/) e [scuole](https://designers.italia.it/modello/scuole/). Questa revisione affronta una serie di sfide peculiari, data la sua applicazione in ambito pubblico:
```

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Update _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md

Co-authored-by: Fabio Bonelli <fb@fabiobonelli.it>
- subject may not be empty
- type may not be empty
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a0e9c2df10c702623ec6f02223c4c5169801862e
</p>

File: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- un design system pubblico non viene applicato da enti tutti uguali tra loro, ma da una serie di enti con le loro specificità, dimensione, ruolo e obiettivi. Diversamente dal privato, la coerenza delle interfacce **deve lasciare un ampio spazio alla personalizzazione, rimanendo nel perimetro dell’usabilità, dell’accessibilità e del rispetto delle norme**.
+
+## Appuntamento al 9 febbraio
+
+Il prossimo appuntamento per la community si terrà il **9 febbraio  alle ore 15.30** e sarà dedicato a queste sfide grazie al contributo di: 
+- **Daniele Tabellini**, UI/UX designer, Dipartimento per la Trasformazione Digitale, che coordinerà nei prossimi mesi il lavoro di aggiornamento del design system del Paese,
+- **Marco Liberati**, il main contributor della libreria Design React Kit, 
+- **Giacomo Grassi**, l’head of customer experience di INPS, che ha da poco affrontato la sua sfida nel lancio di SIrio, il design system dell’ente.
```
Comment (bfabio) [inline _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:28]:
```suggestion
- **Giacomo Grassi**, l’head of customer experience di INPS, che ha da poco affrontato la sua sfida nel lancio di Sirio, il design system dell’ente.
```

File: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md (hunk 2 above)
Comment (Francescopog) [reply _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:13]: Per me va bene come proponi

File: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+subtitle: Quali le sfide da affrontare? Ne parliamo il 9 febbraio!
+date: 2022-01-25 12:00:00 +0200
+tags: developers designers designsystem
+categories: news
+author: Michela Gabrieli
+image: /assets/icons/logo-it.png
+locale: it
+---
+
+Vi siete mai chiesti perché alcuni siti sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo. Per esempio i grandi e-commerce, i social network, il nostro sito preferito di streaming? Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente.
```
Comment (bfabio) [inline _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md:13]:
```suggestion
Vi siete mai chiesti perché alcuni siti, per esempio i grandi e-commerce o il vostro sito preferito di streaming, sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo?
Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente. 
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**